frontend/src/pages/api/feedback/index.ts

```diff
@@ -0,0 +1,107 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
Trengs det å bruke Next.js api som mellomledd her? Kan vi ikke bare sende request rett fra Next.js enten client- eller server-side? Var vel det som ble gjort tidligere.

Reply:
Jeg måtte gjøre det for å få tilgang til `sesssion.tokenId`.

Reply:
Aha skjønner, siden backend sjekker JWT.

Reply:
Yup, backend sjekker JWT og om eposten i tokenen er i listen.